FirebaseFirestoreTestingSupport.podspec

```diff
@@ -0,0 +1,63 @@
+Pod::Spec.new do |s|
```

Comment:
If the podspec is only for testing and not for publishing, it might be better to organize in a lower level directory like how Firestore manages LibFuzzer and GoogleBenchmark in Firestore/Example.

Reply:
I like the idea! Let me try to restructure it to see if it works well.